app/models/agents/java_script_agent.rb

```diff
@@ -102,6 +102,7 @@ def execute_js(js_function, incoming_events = [])
           memory.to_json
         end
       end
+      context["unescapeHTML"] = lambda { |a, x| CGI.unescapeHTML(x) }
```

Comment:
I'd probably call it `unescapeHtml`.  I assume JavaScript's existing [`unescape()`](http://www.w3schools.com/jsref/jsref_unescape.asp) doesn't work?